### PR TITLE
[Torrentland] Update categories

### PR DESCRIPTION
**Pull request by RicardoVelaC**
Adds new Movies/HD category for music HD concerts

That fix latest errors in log when search in Torrentland

### CODE REVIEW

File: src/Jackett.Common/Definitions/torrentland.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ caps:
     - {id: 32, cat: Movies/DVD, desc: "Peliculas - DVDFULL"}
     - {id: 14, cat: Movies/SD, desc: "Peliculas - HDRip"}
     - {id: 15, cat: Movies/DVD, desc: "Peliculas - DVDRip"}
+    - {id: 88, cat: Movies/HD, desc: "Videos Musicales"}
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/torrentland.yml:28]:
```suggestion
    - {id: 88, cat: Movies/HD, desc: "Peliculas - Videos Musicales"}
```

**Review by ilike2burnthing — CHANGES_REQUESTED** [review]

**Comment by ilike2burnthing** [comment]
Would a more accurate category be `Audio/Video` rather than `Movies/HD`?

Categories can be found here - https://github.com/Jackett/Jackett/wiki/Jackett-Categories

File: src/Jackett.Common/Definitions/torrentland.yml (hunk 1 above)
Comment (RicardoVelaC) [reply src/Jackett.Common/Definitions/torrentland.yml:28]:
```suggestion
    - {id: 88, cat: Audio/Video, desc: "Peliculas - Videos Musicales"}
```

Yes, it's better

**Comment by ilike2burnthing** [comment]
Thanks - [v0.17.449](https://github.com/Jackett/Jackett/releases/tag/v0.17.449)

**Pull request merged**